code/modules/surgery/organs/lungs.dm

```diff
@@ -406,6 +406,14 @@
 	safe_toxins_min = 16 //We breath THIS!
 	safe_toxins_max = 0
 
+/obj/item/organ/lungs/oozeling
+	name = "oozling vacuole"
+	desc = "A large organelle designed to store oxygen and filter plasma."
+
+	safe_oxygen_min = 16 //We breath THIS!
+	safe_toxins_min = 0 //We dont't breath this
+	safe_toxins_max = 0
```

Comment:
These are the default values for lungs, this won't change anything.

Reply:
Will test when im home from work, pretty sure these are the same as the plasma lungs

Reply:
tested, 0/0 works perfectly for whats needed

Reply:
Default values are
```dm
	var/safe_toxins_min = 0
	var/safe_toxins_max = 0.05
```

0 disables the check entirely, meaning they can breath in toxic and non-toxic atmospheres without a respirator.